README.md

```diff
@@ -58,4 +110,159 @@ are empty or missing, dashboard is effectively disabled.
+   likely to happen during this stage. Use to set up a list of deliverables for an activity and a deadline to upload 
+   them. If deliverable templates are provided to students, the best place to place them is stages of this type.
+* `Team Evaluation` - allows students to provide anonymous feedback to teammates. Intended use is to allow students
+    in the team provide and receive feedback about their team work, role in the team, communication skills, etc.
+* `Peer Grading` - this stage, in addition to `Submission` stage forms a backbone of group project. During this stage
+    deliverables uploaded earlier are graded by either students in other groups or Teaching Assistants. Grading criteria
+    can be delivered to students as contents of this stage or resources on this stage.
+* `Evaluation Display` - this stage allows students review their teammate feedback. Use with `Team Evaluation` stage.
+* `Grade Display` - this stage allows students review their group grade and feedback received from graders. Use with 
+    `Peer Grading` stage.
+
+Detailed list of available XBlocks are [available in the docs folder](/docs/XBlocks.md).
+
+Group Project XBlock v2 uses standard Studio editing capabilities:
+
+<img src="/docs/images/studio_edit1.png" width="400" alt="Studio screenshot displaying project navigator XBlock, with outlined "View" (red outline) and "Edit" (blue outline) buttons">
+
+* Edit button (blue outline) is used to edit XBlock settings, i.e. "Display name", "Open Date", "Close Date", etc.
+* View button (red outline) allows to "jump into" an XBlock and view it's nested XBlocks, allowing to add, configure, 
+    reorder and delete them.
+
+This screenshot shows the same Group Project Navigator XBlock "from the inside", displaying Project Navigator Views
+added to the navigator. 
+
+<img src="/docs/images/studio_edit2.png" width="400" aria-hidden="true">
+
+## Apros configuration
+
+See corresponding document in Apros (TBD)
+
+# Development
+
+Short summary: developing on Group Project XBlock v2 and running tests requires, at the minimum:
+
+* python>=2.7,<3.0
+* pip>=6.0
+* node.js>=0.10
+* npm>=1.3.10 (might work with older versions, but not checked)
+
+Other dependencies are installed via `pip install -r requirements/dev.txt` and `npm install`.
+
+Group Project XBlock v2 contains a Makefile to help with most common operations.
+
+Running tests:
+
+    * ./run_tests.py to run all python tests (integration tests are run in actual firefox window)
+    * `xvfb-run --server-args="-screen 0, 1920x1080x24" ./run_tests.py` - runs all python tests in virtual X server.
+    * `./node_modules/.bin/karma start tests/js/karma.conf.js` - run JS tests in continuous mode (stays open, 
+        watches file changes, re-runs the suite on file change)
+    * `./node_modules/.bin/karma start tests/js/karma.conf.js --single-run` - run JS tests once.
```

Comment:
@e-kolpakov The rendered Markdown for these items looks a bit weird. I think you can fix that by removing the leading spaces before the items. "run_tests.py" in the first item is missing backticks.